apiary.apib

```diff
@@ -0,0 +1,515 @@
+                    "Logo": "BHP",
+                    "DeletedAt": "2021-04-25T11:55:34Z",
+                    "CreatedAt": "2021-04-25T11:55:34Z",
+                    "UpdatedAt": "2021-04-25T11:55:34Z"
+                    }],            
+                "buyerID": 1
+                },
+             {
+                "buyer": [
+                {
+                    "Name": "BMC",
+                    "ABN": "77 123 765 890",
+                    "Logo": "BMC",
+                    "DeletedAt": "2021-04-25T11:55:34Z",
+                    "CreatedAt": "2021-04-25T11:55:34Z",
+                    "UpdatedAt": "2021-04-25T11:55:34Z"
+                    }],            
+                "buyerID": 2
+                },    
+             {
+                "buyer": [
+                {
+                    "Name": "BMA",
+                    "ABN": "58 566 908 203",
+                    "Logo": "BMA",
+                    "DeletedAt": "2021-04-25T11:55:34Z",
+                    "CreatedAt": "2021-04-25T11:55:34Z",
+                    "UpdatedAt": "2021-04-25T11:55:34Z"
+                    }],            
+                "buyerID": 3
+                }            
+        ]
+
+### Create a New Buyer [POST]
+
++ Response 201 (application/json)
+
+    + Headers
+ 
+            Location: /Buyers/2
+
+    + Body
+
+            {
+                "question": "New Buyer",
+                "published_at": "2015-08-05T08:40:51.620Z",
+                "buyer": [
+                    {
+                        "Name": "smashers",
+                        "ABN": "61 882 999 855",
+                        "Logo": "pem.png"
+                    }, {
+                        "Name": "Toolbox",
+                        "ABN": "21 222 311 556",
+                        "Logo": "tool.png"
+                    }, {
+                        "Name": "Drillbit",
+                        "ABN": "88 788 311 765",
+                        "Logo": "drill.png"
+                    }, {
+                        "Name": "Hammer",
+                        "ABN": "46 234 567 692",
+                        "Logo": "hammer.png"
+                    }
+                ]
+            }
+
+
+## Profile [/profile]
+
+### Current Profiles [GET]
+
++ Response 200 (application/json)
+
+        [
+            {
+                "userID": 1
+                "user": [
+                {
+                    "Name": "BHP",
+                    "email": "bhp@test.com",
+                    "password": "BHP1234",
+                    "BuyerID": 1,
+                    "SupplierID": ,
+                    "DeletedAt": "2021-04-25T11:55:34Z",
+                    "CreatedAt": "2021-04-25T11:55:34Z",
+                    "UpdatedAt": "2021-04-25T11:55:34Z",
+                    "Token": "Asi120Ke",
+                    "refreshToken": "asdasaccAE"
+                    }]},      
+            {
+                "userID": 2
+                "user": [
```

Comment:
Here again, its not an Array "[" its an object "{", I think it will be good to try to consume using POSTMAN to check if the json you are typing is valid...